pkgs/by-name/re/regclient/package.nix

```diff
@@ -34,30 +35,35 @@ buildGoModule rec {
   ldflags = [
     "-s"
     "-w"
-    "-X github.com/regclient/regclient/internal/version.vcsTag=${tag}"
+    "-X github.com/regclient/regclient/internal/version.vcsTag=${finalAttrs.tag}"
   ];
 
   nativeBuildInputs = [
     installShellFiles
     lndir
   ];
 
-  postInstall = lib.concatMapStringsSep "\n" (bin: ''
-    export bin=''$${bin}
-    export outputBin=bin
+  postInstall = lib.concatMapStringsSep "\n" (
+    bin:
+    ''
+      export bin=''$${bin}
+      export outputBin=bin
```

Comment:
I would expect outputBin to be a nix attribute on the same level as postInstall. I first thought this would be used in bash code and it took me a bit to realise how it really works.

Maybe it is only a concidence that it collides with an attribute on the same level 😅